src/plugins/dynamic_dagrun_operator.py

```diff
@@ -46,14 +44,14 @@ def __init__(
         """
         super().__init__(
             trigger_dag_id=None,
-            execution_date=None,
+            execution_date=execution_date,
```

Comment:
handig ;-)

Reply:
Another manual change that should have been a separate commit. Let me see if I can change that.

Reply:
Done.